<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="document_text_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>631</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="numbers_list_view">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="phone_numbers_btn">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run extraction</string>
    </property>
   </widget>
   <widget class="QLabel" name="phone_numbers_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Extracted phone numbers</string>
    </property>
   </widget>
   <widget class="QLabel" name="email_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>200</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Extracted emails</string>
    </property>
   </widget>
   <widget class="QPushButton" name="email_btn">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run extraction</string>
    </property>
   </widget>
   <widget class="QListView" name="email_list_view">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>250</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="topic_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>380</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Topic of document</string>
    </property>
   </widget>
   <widget class="QPushButton" name="topic_btn">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>380</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run extraction</string>
    </property>
   </widget>
   <widget class="QListView" name="topis_list_view">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>430</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="document_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Document</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1084</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
